cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js

```diff
@@ -66,8 +82,12 @@ window.makeCloudinaryImagesResponsive = function () {
     var $cldEl = $('.cloudinary-data-container');
     var cloudinaryObj = $cldEl.data('cloudinary');
     if ($cldResponsiveImgTags && $cldResponsiveImgTags.length > 0) {
-        window.cldObj = window.cldObj || cloudinary.default.Cloudinary.new({cloud_name: cloudinaryObj.cloudName || cloudinaryObj}); // eslint-disable-line no-undef
-        window.cldObj.responsive(); // eslint-disable-line no-undef
+        if (window.cldObj) {
+            window.cldObj = window.cldObj;
```

Comment:
@asad-rafter I missed this one... Could you simplify the logic of this block?

Reply:
@yuval-cloudinary changes have been made